docs/specs/target.md

```diff
@@ -0,0 +1,157 @@
+    overridable by the user.
+  - SHOULD request 6 GiB of disk space per vCPU (48 GiB in this case), MUST be
+    overridable by the user.
+  - SHOULD use the following instances if deployed in a listed cloud.
+    - AWS SHOULD default to `c6g.2xlarge` instances with 48 GiB of EBS `io2`
+      disk space.
+    - Azure SHOULD default to `f8` instances with 48 GiB of standard SSD disk
+      space.
+    - GCP SHOULD default to `c2` instances with 8 vCPUS, 16 GiB of memory, and
+      48 GiB of SSD persisted disk space.
+- Scaling
+  - MUST deploy along with a load balancer to enable horizontal autoscaling.
+  - Autoscaling SHOULD be enabled by default driven by an 85% CPU utilization
+    target over a rolling 5 minute window.
+  - Autoscaling SHOULD have a stabilization period of 5 minutes.
+
+### 5. Unified Architecture
+
+The [unified architecture][unified_architecture] deploys Vector on each
+individual node for local data collection, as well as onto dedicated nodes for
+data aggregation. The requirements for both the [agent](#3-agent-architecture)
+and the [aggregator](#4-aggregator-architecture) apply to this architecture.
+
+## 6. Hardening
+
+- Setup
+  - An unprivileged Vector service account SHOULD be created upon installation
+    for running the Vector process.
+- Data hardening
+  - Swap SHOULD be disabled to prevent in-flight data from leaking to disk.
+    Swap would also make Vector prohibitively slow.
+  - Vector's data directory SHOULD be read and write restricted to Vector's
+    dedicated service account.
+  - Core dumps SHOULD be prevented for the Vector process to prevent in flight
+    data from leaking to disk.
+- Process hardening
+  - Vector's artifacts
+    - All communication during the setup process, such as downloading Vector
+      artifacts, MUST use encrypted channels.
+    - Downloaded Vector artifacts MUST be verified against the provided
+      checksum.
+    - The latest Vector version SHOULD be downloaded unless otherwise specified
+      by the user.
+  - Vector's configuration
+    - Vector's configuration directory SHOULD be read restricted to Vector's
+      service account.
+  - Vector's runtime
+    - Vector SHOULD be run under an unprivileged, dedicated service account.
+    - Vector's service account SHOULD NOT have the ability to overwrite Vector's
+      binary or configuration files. The only directory the Vector service
+      account should write to is Vector’s data directory.
+- Network hardening
+  - Configured sources and sinks SHOULD use encrypted channels by default.
+
+[agent_architecture]: https://vector.dev/docs/setup/going-to-prod/arch/agent/
+[aggregator_architecture]: https://vector.dev/docs/setup/going-to-prod/arch/aggregator/
+[default_agent_configuration]: https://github.com/vectordotdev/vector/blob/target-specification/config/agent/vector.yaml
+[default_aggregator_configuration]: https://vector.dev/docs/reference/configuration/default/aggregator.yaml
```

Comment:
Not sure where we want the default configs to live, or at least where to link to them.

Reply:
Somewhere in the the `config` directory in the Vector repo. So `config/agent.yaml` and `config/aggregator.yaml`, or `config/defaults/*`/